nav2-usability/scenario_code/lib/nested-routing/nested_routing_beamer.dart

```diff
@@ -21,15 +21,18 @@ class BooksApp extends StatelessWidget {
   Widget build(BuildContext context) {
     return MaterialApp.router(
       title: 'Books App',
-      routerDelegate: BeamerRouterDelegate(
+      routerDelegate: BeamerDelegate(
         initialPath: '/books/new',
         locationBuilder: SimpleLocationBuilder(
           routes: {
-            '/*/*': (context) => HomeScreen(),
+            '*': (context) => BeamPage(
+                  key: ValueKey('${context.currentBeamLocation.state.uri}'),
```

Comment:
Does this new line address a bug in the previous version of this snippet or satisfy a requirement of v0.13?

Reply:
This addresses the bug that was already present, but I didn't notice. Specifically, the bug was that manually typing `/settings` URL didn't work every time. It also looks clearer than somewhat vague usage of `/*/*`